opencog/haskell/OpenCog/AtomSpace/Api.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE ForeignFunctionInterface #-}
+
+module OpenCog.AtomSpace.Api (
+      asAddNode
+    , runOnNewAtomSpace
+    , AtomSpace
+    ) where
+
+-- Note that I don't export the AtomSpace data constructors nor the asDelete/asNew functions.
+
+import Foreign
+import Foreign.C.Types
+import Foreign.C.String
+import Control.Exception
+import OpenCog.AtomSpace.Types
+import Control.Monad.IO.Class
+import Control.Monad.Trans.Reader
+import Data.Functor
+
+-- Internal AtomSpace reference to a mutable C++ instance of the AtomSpace class.
+newtype AtomSpaceRef = AtomSpaceRef (Ptr AtomSpaceRef)
+
+-- Main Data Type for representing programs working on an AtomSpace.
+-- We have to use the IO monad because of the use of FFI for calling c functions for working
```

Comment:
Hi Nil, Yes, I had read this standards before, but I forgot to care about column width! I fixed this in 835fdba7286b2739ee52ea48eb4eff0f3a99e2d5 . I will be more careful.